.github/workflows/build-linux-installer-rpm.yml

```diff
@@ -0,0 +1,175 @@
+        pip3 install setuptools_scm
+        echo "::set-output name=CHIA_INSTALLER_VERSION::$(python3 ./build_scripts/installer-version.py)"
+        deactivate
+
+    - name: Test for secrets access
+      id: check_secrets
+      shell: bash
+      run: |
+        unset HAS_SECRET
+        if [ -n "$SECRET" ]; then HAS_SECRET='true' ; fi
+        echo ::set-output name=HAS_SECRET::${HAS_SECRET}
+      env:
+        SECRET: "${{ secrets.INSTALLER_UPLOAD_SECRET }}"
+
+    - name: Run install script
+      env:
+        INSTALL_PYTHON_VERSION: ${{ matrix.python-version }}
+        BUILD_VDF_CLIENT: "N"
+      run: |
+        sh install.sh
+
+    - name: Setup Node 12.x
+      uses: actions/setup-node@v2.1.5
+      with:
+        node-version: '12.x'
+
+    - name: Build .rpm package
+      run: |
+        . ./activate
+        sudo apt-get -y install rpm
+        ldd --version
+        cd ./chia-blockchain-gui
+        git status
+        cd ../build_scripts
+        sh build_linux_rpm.sh amd64
+
+    - name: Upload Linux artifacts
+      uses: actions/upload-artifact@v2
+      with:
+        name: Linux-Installers
+        path: ${{ github.workspace }}/build_scripts/final_installer/
+
+    - name: Configure AWS Credentials
+      if: steps.check_secrets.outputs.HAS_SECRET
+      uses: aws-actions/configure-aws-credentials@v1
+      with:
+        aws-access-key-id: ${{ secrets.INSTALLER_UPLOAD_KEY }}
+        aws-secret-access-key: ${{ secrets.INSTALLER_UPLOAD_SECRET }}
+        aws-region: us-west-2
+
+    - name: Upload to s3
+      if: steps.check_secrets.outputs.HAS_SECRET
+      env:
+          CHIA_INSTALLER_VERSION: ${{ steps.version_number.outputs.CHIA_INSTALLER_VERSION }}
+      run: |
+          ls ${{ github.workspace }}/build_scripts/final_installer/
+          aws s3 cp ${{ github.workspace }}/build_scripts/final_installer/*.rpm s3://download-chia-net/builds/
+#          aws s3 cp ${{ github.workspace }}/build_scripts/final_installer/*.deb s3://download-chia-net/builds/
+
+    - name: Create Checksums
+      if: startsWith(github.ref, 'refs/tags/')
+      env:
+        CHIA_INSTALLER_VERSION: ${{ steps.version_number.outputs.CHIA_INSTALLER_VERSION }}
+      run: |
+         ls ${{ github.workspace }}/build_scripts/final_installer/
+         sha256sum ${{ github.workspace }}/build_scripts/final_installer/chia-blockchain-${{ steps.version_number.outputs.CHIA_INSTALLER_VERSION }}-1.x86_64.rpm > ${{ github.workspace }}/build_scripts/final_installer/chia-blockchain-${{ steps.version_number.outputs.CHIA_INSTALLER_VERSION }}-1.x86_64.rpm.sha256
+         ls ${{ github.workspace }}/build_scripts/final_installer/
+#         sha256sum ${{ github.workspace }}/build_scripts/final_installer/chia-blockchain_${{ steps.version_number.outputs.CHIA_INSTALLER_VERSION }}_amd64.deb > ${{ github.workspace }}/build_scripts/final_installer/chia-blockchain_${{ steps.version_number.outputs.CHIA_INSTALLER_VERSION }}_amd64.deb.sha256
```

Comment:
nit: remove commented lines in this file pertaining to deb and commented lines in the other file pertaining to rpm?

Reply:
Thanks. This is still a work in progress and not ready for review yet.